plugins/push/frontend/public/stylesheets/main.css

```diff
@@ -987,7 +1015,7 @@ table.d-table .nothing { opacity: 0.3; }
 
 .comp-push input[type="number"]::-webkit-inner-spin-button,
 .comp-push input[type="number"]::-webkit-outer-spin-button {
-    -webkit-appearance:none;
```

Comment:
did you have time to check functionality? I couldn't find an answer for the problem right above. So I can cancel to adding a spinner to input.